config/vehicle-channel-config.js

```diff
@@ -0,0 +1,13 @@
+var SettingsLoader=require('../app/util/PersistentSettings');
+
+var vehicle_channel_config = {
```

Comment:
This file is too speficic. Let's generalize it to just vehicle_config for vehicle all vehicle specific settings